src/Fixer/Whitespace/IndentationTypeFixer.php

```diff
@@ -120,7 +120,7 @@ private function fixIndentToken(Tokens $tokens, $index)
         $content = $tokens[$index]->getContent();
         $previousTokenHasTrailingLinebreak = false;
 
-        // @TODO 3.0 this can be removed when we have a transformer for "T_OPEN_TAG" to "T_OPEN_TAG + T_WHITESPACE"
+        // @TODO this can be removed when we have a transformer for "T_OPEN_TAG" to "T_OPEN_TAG + T_WHITESPACE"
```

Comment:
maybe the comment can be removed completely as I don't think we will do this any time soon 
(I tried and it has a lot more of catches/issues to pull off than just this one commented here)

Reply:
to be honest - I don't know.
I would love to give it separated thought in one of the sessions, but not a prio for now.

for that, let's keep it as-is, simply dropping the 3.0 reference